src/argilla/client/feedback/integrations/config.py

```diff
@@ -0,0 +1,34 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from dataclasses import dataclass
+
+from argilla.utils.dependency import require_version
+
+
+def _check_if_installed(dependency: str) -> None:
+    try:
+        require_version(dependency)
+        return True
+    except Exception as e:
+        return False
```

Comment:
Because using the decorators implies that the dependency check is ran every time the function is called, while keeping everything in the config can give us more information on the required dependencies for the existing integrations, and that's just ran once

Reply:
But yes, I agree that's better to keep using `requires_version` for the moment!